addon/mixins/in-viewport.js

```diff
@@ -3,53 +3,62 @@
 
 import Ember from 'ember';
 
-var on           = Ember.on;
-var get          = Ember.get;
-var Mixin        = Ember.Mixin;
-var debounce     = Ember.run.debounce;
+var on = Ember.on;
```

Comment:
I've reformatted only variables at the beginning of the file. But when you format variable by the equal sign, this is the price you pay, when you add a new variable. Then you have to reformat whole column.  Same goes for formatting by : sign. when you add variable with the longer name then you again will have a PR where whole column is changed. If you insist, I can change it.